components/EventForm.tsx

```diff
@@ -21,13 +23,41 @@ export default function EventForm({ event, onSubmitClick, formAction }: EventFor
           title: '',
           page: '',
           type: '',
+          track: '',
           location: '',
           speakers: [],
           startDate: new Date(),
           endDate: new Date(),
           Event: -1,
         },
   );
+
+  const { user } = useAuthContext();
+  const [loading, setLoading] = useState(true);
+  const [errors, setErrors] = useState<string[]>([]);
+  const [tracks, setTracks] = useState<string[]>([]);
+
+  useEffect(() => {
+    async function getTracks() {
+      try {
+        const { status, data } = await RequestHelper.get<[ScheduleEvent]>(`/api/schedule`, {
+          headers: {
+            Authorization: user.token!,
+          },
+        });
+        const tracks = data.map((event) => event.track);
+        const uniqueTracks = new Set(tracks);
+        setTracks(Array.from(uniqueTracks));
+      } catch (error) {
+        console.error(error);
+        setErrors((prev) => [...prev, 'Unexpected error. Please try again later']);
```

Comment:
Is there a way for us to know what caused the error here? Since this is used by a lot of people I think we should aim to have specific error messages with as much precise information as possible.